package.json

```diff
@@ -84,6 +86,7 @@
   "dependencies": {
     "chalk": "^2.3.2",
     "debug": "^3.1.0",
+    "detect-installed": "^2.0.4",
```

Comment:
I do not see where this is `require`d in this PR. Is it needed?

Reply:
good catch, thanks!